spec/idMaker.js

```diff
@@ -0,0 +1,9 @@
+const generateId = (userContext, events, done) => {
+  const id = Math.ceil(Math.random() * 10000000);
+  userContext.vars.id = id;
+  return done();
+};
+
+module.exports = {
```

Comment:
Not a requirement by any means, but if you want to you can just set the entire function to `module.export = (userContext, events, done) => ...` if you want to to make this even more concise. But it's very readable as is.